t/unittest/UnifiedApi/Controller/Config/Roles.t

```diff
@@ -37,6 +38,23 @@ my $collection_base_url = '/api/v1/config/roles';
 
 my $base_url = '/api/v1/config/role';
 
+{
+    my $id = "test_role_${$}_1";
+    my $acl = <<ACL;
+permit ip 172.16.1.0 0.0.0.255 host 192.168.3.154
+ACL
+    $t->post_ok($collection_base_url => json => { id => $id, acls => $acl })
+      ->status_is(201);
+}
+
+{
```

Comment:
I think it will be important to test the various cases where there should be warnings (ACL too large, equipment doesn't support it, etc)

Reply:
Ok, I would need to add the equipment does not support it.
I already have the ACL to large check.